README.md

```diff
@@ -154,7 +153,23 @@ On Linux find executable file `./darknet` in the root directory, while on Window
 
 * Yolo v3 COCO-model: `darknet.exe detector demo data/coco.data yolov3.cfg yolov3.weights http://192.168.0.80:8080/video?dummy=param.mjpg -i 0`
 
-### How to compile on Linux
+### How to compile on Linux (using `cmake`)
+
+The `CMakeLists.txt` will attempt to find installed optional dependencies like
+CUDA, cudnn, ZED and build against those. It will also create a shared object
+library file to use `darknet` for code development.
+
+Inside the cloned repository:
```

Comment:
May be better?

`Do inside the cloned repository:`

Reply:
Noted and resolved in latest commit